libs/@local/hash-isomorphic-utils/src/save.ts

```diff
@@ -390,18 +384,19 @@ export const save = async (
 
   // const entityTypeForComponentId = new Map<string, string>();
 
-  const [actions, placeholderToDraft] = await calculateSaveActions(
+  const [actions, placeholderToDraft] = calculateSaveActions(
     store,
     ownedById,
     /** @todo This type ID should *not* be hardcoded as is here. */
     TEXT_ENTITY_TYPE_ID,
```

Comment:
Does anyone know what this `@todo` is for? Not clear if it's still relevant, or why we don't just use `TEXT_ENTITY_TYPE_ID` in `calculateSaveActions` directly

probably one for @thehabbos007 @nathggns

Reply:
What is the problem in hardcoding it like this? What is the thing we want to be doing instead?

Reply:
Thank you – I will update this todo to a comment explaining that in #2503